unit/java_bytecode/java_bytecode_parse_generics/Bar.java

```diff
@@ -0,0 +1,2 @@
+class Bar extends Foo
+{}
```

Comment:
Nit: this file (and others) are missing a new line at the end of the file (and others). You can make sure CLion inserts these using Settings > Editor > General > Ensure Line Feed at end of file on save